### PR TITLE
Adding an entry for checking registry quota usage

### PR DESCRIPTION
**Pull request by SriniBanala**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -82,6 +82,30 @@ az acr repository show-manifests -n myRegistry --repository myRepository --query
 
 Note: You can add `-y` in the delete command to skip confirmation
 
+## I deleted images to free up registry quota usage but the available quota doesn't get updated
+
+This can happen if the underlying layers are still being referenced by other container images. If you delete an image with no references, the registry usage will be updated after a few minutes. Here are the steps for verifying the quota usage.
+
+        Download a test image from DockerHub
```
Comment (ehotinger) [inline docs/FAQ.md]: should fix the formatting, it's garbled; use `sh` or similar as syntax hightlighting for the code blocks

Comment (sajayantony) [reply docs/FAQ.md]: Avoid indenting blocks here.

Comment (SriniBanala) [reply docs/FAQ.md]: Thanks. Edited in VS to fix the formatting.

**Comment by sajayantony** [comment]
This is asking a user to download a random image from docker hub and update to their registry. Would strongly avoid recommending that.

Could you rephrase this to something like this ? 

---

## How do I validate storage quota changes?

Create an image with a 1GB layer using the following docker file. This ensures that the image has has a layer that is not shared by any other image in the registry. 

```dockerfile
FROM alpine
RUN dd if=/dev/urandom of=1GB.bin  bs=32M  count=32
RUN ls -lh 1GB.bin
```

Build and push the image to your registry using the `docker` CLI.
```
docker build -t myregistry.azurecr.io/1gb:latest .
docker push myregistry.azurecr.io/1gb:latest
```
You should be able to see that the storage used, has increased in the portal or you can query usage using the CLI.

```
az acr show-usage -n myregistry
```

Delete the image using the Azure CLI or portal and check the updated usage in a few minutes. 
```
az acr repository delete -n myregistry --image 1gb
```

**Comment by sivagms** [comment]
i'm fine with the approach of asking user to build the image.

**Comment by SriniBanala** [comment]
@sajayantony  Looks much better. Thanks Sajay.
